Website/src/items/content/EdgeWebsite/Data/CMP Data/Testimonies/Striva.yml

```diff
@@ -23,6 +23,9 @@ Languages:
     - ID: "30de01d7-233e-42ac-bb14-751e9d60b1cd"
       Hint: Provider
       Value: Striva
+    - ID: "32fcc3c3-e6bb-4376-8bd8-bddb62fedb57"
+      Hint: LogoTransformation
+      Value: 
```

Comment:
Leaving the field in the serialized item will make the transformation field value to be an empty string instead of the standard value. We must remove the whole 3 lines to get back to the standard values value.

```
    - ID: "32fcc3c3-e6bb-4376-8bd8-bddb62fedb57"
      Hint: LogoTransformation
      Value: 
```

Reply:
You can revert all changes to this item.